test/util.test.js

```diff
@@ -66,6 +66,18 @@ describe('util.removeUndefined', function () {
     var q4 = {where: {x: 1, y: date}};
     should.deepEqual(removeUndefined(q4), {where: {x: 1, y: date}});
 
+    // test handling of undefined
+    var q5 = {where: {x: 1, y: undefined}};
+    should.deepEqual(removeUndefined(q5, 'setNull'), {where: {x: 1, y: null}});
+
+    var q6 = {where: {x: 1, y: undefined}};
+    try {
+      removeUndefined(q6, 'throwError');
+    }
+    catch(err) {
```

Comment:
BTW the canonical way of testing this is to use `should.throw`, see [should.js docs](https://github.com/tj/should.js/#throw-and-throwerror). It's also better to assert that the error message thrown loosely contains the expected error message.

```js
(function(){ removeUndefined(q6, 'throwError') }).should.throw(/`undefined` in query/);
```